resources/lib/playerinfo.py

```diff
@@ -17,30 +17,36 @@ def __init__(self, **kwargs):
         self._tracker = None
         self._last_pos = 0
         self._total = None
+        self._stop = False
         Player.__init__(self)
-        while not self._monitor.abortRequested():
-            if self._monitor.waitForAbort(10):
+        while not self._monitor.abortRequested() and not self._stop:
+            if self._monitor.waitForAbort(1):
                 break
+        self._info((self._last_pos, self._total))
 
     def onAVStarted(self):  # pylint: disable=invalid-name
         ''' called when Kodi has a video or audiostream '''
         self._total = self.getTotalTime()
         self._tracker = Thread(target=self.stream_position)
+        self._tracker.name = 'VRTNUThread'
         self._tracker.start()
+        self._tracker.name = 'VRTNUThread'
```

Comment:
Naming this Thread once is enough.

Reply:
It doesn't work, so I was trying it once before and after the start. This PR is still a work in progress :-/

Reply:
Strange...